app/components/DataCardInput.tsx

```diff
@@ -16,7 +16,7 @@ interface DataCardInputProps {
   /**
    * Controls the type of input to be rendered
    */
-  inputType?: InputType;
+  inputType?: InputType; // wondering if this shoudl be required and not optional
```

Comment:
if you dont specify an inputType, the form on the data card input component only shows the save and cancel buttons, there is no default input option. Which makes me think this should be required, or do we have a fallback input?

Reply:
I see what you mean :/ 

I think ideally the type definition would require `inputType` if `editable` is `true`. Are you up for reworking the `ColDataCardDef` in this PR?

Reply:
I will look at reworking the `ColDataCardDef`, but DataCardInput doesn't render unless a column is editable, right?